auth-web/src/components/datatable/components/HeaderFilter.vue

```diff
@@ -71,15 +71,15 @@ export default defineComponent({
       sortedItems: [...props.sortedItems]
     }) as unknown) as BaseTableStateI
 
-    const serverSideFilter = async (header: BaseTableHeaderI) => {
+    const serverSideFilter = debounce(async (header: BaseTableHeaderI) => {
       props.setFiltering(true)
       await header.customFilter.filterApiFn(header.customFilter.value)
       props.setFiltering(false)
-    }
+    })
 
     const filter = async (header: BaseTableHeaderI) => {
       if (header.customFilter.filterApiFn) {
-        _.debounce(serverSideFilter(header), 500)
+        await serverSideFilter(header)
```

Comment:
This looks pretty good, thanks for fixing that!

Reply:
it was really weird it wouldn't be called at all with async no matter how i tried with the lodash library